accessibility/common-remediations.md

```diff
@@ -195,6 +195,56 @@ We don’t do this:
 <a href="knitting.html" target="_blank">All about Knitting</a>
 ```
 
+### No accessible text on controls
```

Comment:
Need to link to this section from the top

Reply:
oops, good spot!